Classes/CRON/FormBuilder/Validation/Validator/FormBuilderValidator.php

```diff
@@ -45,8 +45,11 @@ protected function isValid($value)
 
         foreach ($value as $index => $collectionElement) {
 
-            $node = $this->nodeDataRepository->findOneByIdentifier($index,
-                $this->siteService->getSiteNode()->getWorkspace());
+            if($this->nodeDataRepository->findOneByIdentifier($index, $this->siteService->getSiteNode()->getWorkspace())) {
```

Comment:
what is the reason for this change? And why call the same (potentially expensive) operation twice? Previously it was assigned to `$node`, simply check for `if ($node)` if required.